compiler/src/dotty/tools/dotc/parsing/Parsers.scala

```diff
@@ -1555,8 +1555,7 @@ object Parsers {
         if in.token == LBRACE || in.token == INDENT then
           t
         else
-          if sourceVersion.isAtLeast(future) then
```

Comment:
I'm not 100% sure if this can just be removed or needs to be set to an older source version?